config/webpack.config.dev.js

```diff
@@ -132,9 +132,9 @@ module.exports = {
   // assume a corresponding global variable exists and use that instead.
   // This is important because it allows us to avoid bundling all of our
   // dependencies, which allows browsers to cache those libraries between builds.
+
+  // Since its a local dev build we don't need to mark anything external.
```

Comment:
Like commented above, not bundling all of our dependencies allow browsers to cache those libraries and may make the bundling faster?

Reply:
Sorry didn't mean to overwrite. Since we modified external thought this might be an issue. Have reverted back.